source/config/localhost_config.json

```diff
@@ -7,5 +7,10 @@
        "server_cert": "",
        "server_key": "",
        "root_cert": ""
+    },
+    "core_configuration": {
```

Comment:
the default values should be 0

Reply:
core indexing starts from 0 so we can't set default value as 0, hence setting it to -1 which indicates that any available core can be used and cpu pinning is not needed.